### DISCUSSION TITLE
Will quasar support .AAB soon?

### DISCUSSION THREAD

**Discussion by tgarif** [quasarframework/quasar#10323] [CLI - Capacitor / Cordova modes] [answered]
Beginning from the coming weeks or months, Google Play is not going to support .APK files. We need to generate .AAB files. Does quasar has such kind of opportunity?

**Comment by MilosPaunovic** [comment, marked as answer]
Check this out:
https://cordova.apache.org/docs/en/10.x/guide/platforms/android/#using-flags
